MAUI/ProgressBar/Appearance.md

```diff
@@ -0,0 +1,390 @@
+---
+layout: post
+title: Appearance in .NET MAUI ProgressBar control | Syncfusion
+description: Learn here all about appearance support in Syncfusion .NET MAUI ProgressBar control, its elements and more.
+platform: MAUI
+control: ProgressBar
+documentation: ug
+---
+
+# Appearance in .NET MAUI ProgressBar (Progress Bar)
+
+## Angle
+
+The appearance of the circular progress bar can be customized to semi-circle, arc, and more. The start and end angles can be customized using the `StartAngle` and `EndAngle` properties respectively.
+
+The following code sample demonstrates how to change the appearance of the circular progress bar to semi-circle.
+
+{% tabs %} 
+
+{% highlight xaml %}
+
+<progressBar:SfCircularProgressBar Progress="75" 
+                                   StartAngle="180" 
+                                   EndAngle="360" />
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCircularProgressBar circularProgressBar = new SfCircularProgressBar();
+circularProgressBar.Progress = 75;
+circularProgressBar.StartAngle = 180;
+circularProgressBar.EndAngle = 360;
+this.Content = circularProgressBar;
+
+{% endhighlight %}
+
+{% endtabs %} 
+
+![.NET MAUI Circular Progress Bar with angle customization](images/appearance/angle.png)
+
+## Range colors
+
+Visualize multiple ranges with different colors mapped to each range to enhance the readability of progress.
+
+The colors can be mapped to the specific ranges using the `GradientStops` property, which holds a collection of `ProgressGradientStop`. 
+
+The following properties in the `GradientStops` are used to map the colors to a range:
+
+* `Color:` Represents the color to the specified range.
+* `Value:` Represents the start or end value for the specified color.
+
+The following code sample demonstrates how to map the solid color range in the progress bar.
+
+{% tabs %} 
+
+{% highlight xaml %}
+
+<!--Using the linear progress bar-->
+
+<progressBar:SfLinearProgressBar Progress="100">
+    <progressBar:SfLinearProgressBar.GradientStops>
+        <progressBar:ProgressGradientStop Color="#00bdaf" Value="0"/>
+        <progressBar:ProgressGradientStop Color="#00bdaf" Value="25"/>
+        <progressBar:ProgressGradientStop Color="#2f7ecc" Value="25"/>
+        <progressBar:ProgressGradientStop Color="#2f7ecc" Value="50"/>
+        <progressBar:ProgressGradientStop Color="#e9648e" Value="50"/>
+        <progressBar:ProgressGradientStop Color="#e9648e" Value="75"/>
+        <progressBar:ProgressGradientStop Color="#e9648e" Value="75"/>
```

Comment:
You have provided the same color for the last two segments. Please change it to the circular progress bar.

Reply:
done